[2025-01-30T10:26:00.232+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T10:26:00.266+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-09-02T00:00:00+00:00 [queued]>
[2025-01-30T10:26:00.278+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-09-02T00:00:00+00:00 [queued]>
[2025-01-30T10:26:00.279+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T10:26:00.299+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upload_to_cloud> on 2020-09-02 00:00:00+00:00
[2025-01-30T10:26:00.310+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=386) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T10:26:00.312+0000] {standard_task_runner.py:72} INFO - Started process 394 to run task
[2025-01-30T10:26:00.317+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fhv_to_gcp_dag', 'upload_to_cloud', 'scheduled__2020-09-02T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/homework_fhv_dag.py', '--cfg-path', '/tmp/tmpah25d9ki']
[2025-01-30T10:26:00.319+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask upload_to_cloud
[2025-01-30T10:26:00.432+0000] {task_command.py:467} INFO - Running <TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-09-02T00:00:00+00:00 [running]> on host 9573d401388f
[2025-01-30T10:26:00.593+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fhv_to_gcp_dag' AIRFLOW_CTX_TASK_ID='upload_to_cloud' AIRFLOW_CTX_EXECUTION_DATE='2020-09-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2020-09-02T00:00:00+00:00'
[2025-01-30T10:26:00.594+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T10:26:02.479+0000] {logging_mixin.py:190} INFO - The file /opt/***/data/fhv_2020-09_data.parquet has been uploaded to nrabbi-33-dtc-de-data-bucket/<Blob: nrabbi-33-dtc-de-data-bucket, /data/fhv_2020-09_data.parquet, 1738232762508667>
[2025-01-30T10:26:02.479+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-30T10:26:02.488+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T10:26:02.489+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fhv_to_gcp_dag, task_id=upload_to_cloud, run_id=scheduled__2020-09-02T00:00:00+00:00, execution_date=20200902T000000, start_date=20250130T102600, end_date=20250130T102602
[2025-01-30T10:26:02.532+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T10:26:02.546+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-30T10:26:02.547+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
